prow/e2e-suite-rbac-auth.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Copyright 2017 Istio Authors
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+
+#######################################
+#                                     #
+#             e2e-suite               #
+#                                     #
+#######################################
+
+# Exit immediately for non zero status
+set -e
+# Check unset variables
+set -u
+# Print commands
+set -x
+
+echo 'Running e2e with rbac, with auth Tests'
+./prow/e2e-suite.sh --rbac_enable --auth_enable
```

Comment:
No please keep e2e.sh and remote e2e-suite as e2e.sh is used in pilto and mixer. https://github.com/istio/pilot/blob/master/prow/pilot-e2e-smoketest.sh#L57